README.Rmd

```diff
@@ -32,6 +38,91 @@ You can install the development version of ghqc from [GitHub](https://github.com
 pak::pak("a2-ai/ghqc")
 ```
 
-## Short Tour
+## Interacting with ghqc
+
+``` r
+library(ghqc)
+```
+
+## Configuration Information Repository
+
+The ghqc ecosystem has been developed with the intent organizations can customize elements of the QC process. Currently, the customizable elements are:
+
+- **Checklists**: yaml files contained within a directory named "checklist" *(required)*
+
+- **Prepended Note**: A note to prepend to all checklists named "note" *(optional)*
+
+- **Logo**: A logo to be included in the header of the resultant record's header named as "logo.png"
+
+
+An example structure of the configuration information repository can be found at [github.com/A2-ai/ghqc.example_info_repo](https://github.com/A2-ai/ghqc.example_info_repo)
+
+## ghqc Setup
+
+ghqc has 3 main requirements that must be met before running any of the ghqc ecosystem apps:
+
+1. An environmental variable, `GHQC_INFO_REPO`, must be set to the configuration information repository url in "~/.Renviron". 
+
+2. The configuration information repository must be downloaded to a local location. 
+
+3. All of the dependency packages for the ghqc shiny apps, including `ghqc.app` must be located in the same directory. 
+
+Within this package, a function `ghqc::setup_ghqc()` aids in this setup.
+
+``` r
+ghqc::setup_ghqc()
+```
+
+1. **Renviron Setup**
+
+Enter the url of the configuration information repository described above
+
+![](man/figures/renviron_setup.png)
+
+2. **Configuration Information Download and Verification**
+
+The configuration information will default to installing to "~/.local/share/ghqc/<repo_name>". To download to the default location, which is highly recommended, hit `ENTER`. Otherwise, type in the path in which to download the repository.
+
+In addition to downloading the repository, or remote updates if the repository was previously downloaded, the structure of the repository and the checklist yamls are verified.
```

Comment:
Maybe we can say something more clearly here about how ghqc will check for remote updates and let the user update the local directory if there any (with a ss)

Reply:
https://github.com/A2-ai/ghqc/blob/b22c559d5c7ec223217c29fb4be56437a933fce8/README.Rmd#L86-L92

Changed functionality of `ghqc::setup_ghqc()` to make this more explicit to the user. Also added this section to the README to explicitly discuss update mechanism